.ci/scripts/send-pr.sh

```diff
@@ -8,8 +8,9 @@ readonly DO_SEND_PR=${3:-"true"}
 readonly APM_AGENT_REPO_NAME="apm-agent-${APM_AGENT}"
 readonly GIT_DIR=".ci/git"
 readonly APM_AGENT_REPO_DIR="${GIT_DIR}/${APM_AGENT_REPO_NAME}"
+readonly GITHUB_token="${GITHUB_TOKEN:?"missing GITHUB_TOKEN"}"
 
-git clone "https://github.com/elastic/${APM_AGENT_REPO_NAME}" "${APM_AGENT_REPO_DIR}"
+git clone "https://${GITHUB_token}@github.com/elastic/${APM_AGENT_REPO_NAME}" "${APM_AGENT_REPO_DIR}"
```

Comment:
I don't think you need this

Reply:
Even with the `set -u`?

Reply:
I think you can revert the changes in this file. IIRC, the `-u` is for the undefined variables, in this case, the `GITHUB_TOKEN` is set previously with the `withCredentials`

Reply:
Done!